### PR TITLE
Continue docstring after newline

### LINKED ISSUE #35: Generates doc comments that result in syntax errors
I'm using kopium to generate type definitions for cert-manager's CRDs (https://github.com/jetstack/cert-manager/releases/download/v1.7.1/cert-manager.crds.yaml), and when using `--docs`, it generates doc comments that have syntax errors in them. The generated file is here: https://gist.github.com/antifuchs/533118428707296e1504e2215c77ecb8 - note line [188](https://gist.github.com/antifuchs/533118428707296e1504e2215c77ecb8#file-cert_manager-rs-L188), which is not prefixed with `///` - I suspect that's due to linebreaks in the documentation.

### PR DESCRIPTION
**Pull request by antifuchs**
This fixes #35 and adds a test to ensure that it is/stays fixed (-:

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -227,7 +227,7 @@ impl Kopium {
         // print doc strings if requested in arguments
         if self.docs {
             if let Some(d) = doc {
-                println!("{}/// {}", indent, d);
+                println!("{}/// {}", indent, d.replace("\n", &format!("\n{}///", indent)));
```
Comment (clux) [inline src/main.rs:230]: Probably need a space here after the ///? Normally things don't open with a space after newlines.

File: justfile [hunk 2]
```diff
@@ -36,5 +36,12 @@ test-argo:
   kubectl apply -f tests/app.yaml
   cargo test --test runner -- --nocapture
 
+test-certmanager:
+  kubectl apply --force-conflicts --server-side -f https://github.com/jetstack/cert-manager/releases/download/v1.7.1/cert-manager.crds.yaml
```
Comment (clux) [inline justfile:40]: Ah, thanks! More tests!

**Review by clux — COMMENTED** [review]
Minor comment, but looks great

File: src/main.rs (hunk 1 above)
Comment (antifuchs) [reply src/main.rs:230]: Funny you should mention it - I first tried it with a space, but the generated code looked like `///  post-\n` - one space too many. But it looks like it's cert-manager's own CRDs that include spaces after the newline. I guess a second space can't hurt, will fix.

**Comment by antifuchs** [comment]
Updated!

**Review by clux — APPROVED** [review]

**Comment by clux** [comment]
Thanks a lot. Before I can merge, are you able to `--signoff` your commits as per the DCO check? Legal requirements.

**Comment by clux** [comment]
Thanks a lot!

**Pull request merged**